--- a/configure.ac
+++ b/configure.ac
@@ -6,7 +6,7 @@
 # ----------------------------------------------------------------------
 
 m4_define([htop_git_version],
-   m4_normalize(m4_esyscmd([git describe --abbrev=7 --dirty --always --tags 2> /dev/null || echo ''])))
+   m4_normalize(m4_esyscmd([if test -d .git; then git describe --abbrev=7 --dirty --always --tags 2> /dev/null || echo ''; fi])))
 m4_define([htop_release_version], [3.5.1])
 
 # ----------------------------------------------------------------------
